### ISSUE TITLE
Feature request: Programmable Drums

### ISSUE THREAD

**Issue by Metalmaniac89** [tuntorius/mightier_amp#36]
Hi,

first of all I really love this app as working with the Nux Mighty Plug is much easier now and much more fun (also supported the developer ;-) )

One thing I am missing for practice sessions is a programmable drum/metronome. For example I would like to be able to program the drums in such a way that after for example 4 bars the tempo increases by a set amount. This feature should be able to increase/decrease the tempo after a set amount of time/bars or in steps calculated from the exercise time. 
Basically a implementation of this metronome for the app: https://indiebubbler.github.io/metro/

Would that be possible to implement?

**Comment by tuntorius** [comment]
This is a very good idea. I'll see if I can do it with the built-in metronome/drum styles. They are played back from the amp though, my app doesn't know the exact point they start/loop so I'm not sure if I'll be able to implement the tempo increase by bars. Maybe only by time.

**Comment by Metalmaniac89** [comment]
Thus is

**Comment by Metalmaniac89** [comment]
Thank you very much.

**Comment by moonbaseDelta** [comment]
By the way, it also would be quite awesome if one could write drums in non-standard time signatures, like 7/8 or 13/16 and mix them together. And then apply speed changes

**Comment by tuntorius** [comment]
Added in v1.0.5
